habinezadalvan/haven
# feature(view single article mockup): user is able to view a single ar…

**Description of Task to be completed?**

- created a responsive view-single-article page

**How should this be manually tested?**

- clone the repo
-make sure you have Live Sass Compiler in your editor
- go the folder directory 
- go in your terminal watch sass to compile into CSS
- open viewSingleArticle.html in your browser

**Any background context you want to provide?**

- used sass, HTML

**What are the relevant pivotal tracker stories?**
no story created

Screenshots (if appropriate)
